PIL/ImageMode.py

```diff
@@ -40,6 +40,7 @@ def getmode(mode):
             _modes[m] = ModeDescriptor(m, bands, basemode, basetype)
         # extra experimental modes
         _modes["LA"] = ModeDescriptor("LA", ("L", "A"), "L", "L")
+        _modes["La"] = ModeDescriptor("La", ("L", "a"), "L", "L")
```

Comment:
I went through the review, hit this, started wondering why we didn't have `RGBa` as a mode here, and then determined that modes are a mess.